tools/docker/syzbot/provision-llvm.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Copyright 2024 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+set -e # exit on any problem
+set -o pipefail
+
+if [ -z "$1" ];then
+    echo "Pass LLVM version in the argument."
+    exit 1
+fi
+
+LLVM_VERSION=$1
+
+url="https://mirrors.edge.kernel.org/pub/tools/llvm/files"
+tmp=$(mktemp -d)
+cd $tmp
+
+# download the archive
+curl -L $url/$LLVM_VERSION.tar.gz -o $LLVM_VERSION.tar.gz
+
+# llvm archive intergrity check
+curl -L -s $url/sha256sums.asc | grep $LLVM_VERSION.tar.gz > $LLVM_VERSION.asc
+sha256sum -c $LLVM_VERSION.asc
+
+# gpg signature import
+curl -L -s $url/$LLVM_VERSION.tar.sign -o $LLVM_VERSION.tar.sign
+keyid=$(gpg --list-packets $LLVM_VERSION.tar.sign | grep -oE "keyid .*$" | awk '{print $2}')
+gpg --keyserver keyserver.ubuntu.com --recv-keys $keyid
+# See https://www.kernel.org/signature.html
+gpg --tofu-policy good $keyid
+
+# decompress & gpg signature verification
+gunzip -d $LLVM_VERSION.tar
+gpg --trust-model tofu --verify $LLVM_VERSION.tar.sign $LLVM_VERSION.tar
+
+# untar binaries to the output
```

Comment:
Ditto.

Reply:
Thanks! PTAL

@a-nogikh , @tarasmadan, @dvyukov , please review as well. As this should close https://github.com/google/syzkaller/pull/5375 PR.

Reply:
Following up